R/drive_file.R

```diff
@@ -7,68 +7,17 @@
 #' @export
 #'
 drive_file <- function(id = NULL, ...) {
-  request <- build_drive_file(id = id, ...)
+  request <- build_request(
+    endpoint = "drive.files.get",
+    token = drive_token(),
+    params = list(...,
```

Comment:
If the user specified `fileId = "abcdef"` in `...`, would `build_request()` catch it? Are you checking for params that appear more than once?